README.md

```diff
@@ -199,6 +184,8 @@ You can specify either the tag or the digest.
 
 #### Syntax
 Username and password are optional because the mixin will default to using environment variables provided by DOCKER_USERNAME and DOCKER_PASSWORD from a parameter or a credential.
+See an [example] (/examples/docker-mixin-test/README.md#notes-on-docker-login) for how to use 
```

Comment:
Oops, you have a space between the [] and () that breaks the link

Reply:
my bad just fixed that!